zerver/tests/test_auth_backends.py

```diff
@@ -2336,10 +2336,9 @@ def test_id_token_with_invalid_aud_sent(self) -> None:
             id_token=self.generate_id_token(account_data_dict, audience='com.different.app'),
         )
 
-        with self.apple_jwk_url_mock(), mock.patch('logging.info') as mock_info:
+        with self.apple_jwk_url_mock(),  self.assertLogs(level='INFO') as m:
             result = self.client_get(url, **headers)
-            mock_info.assert_called_once_with('/complete/apple/: %s',
-                                              'Authentication failed: Token validation failed')
+        self.assertEqual(m.output, ["INFO:root:/complete/apple/: {}".format("Authentication failed: Token validation failed")])
```

Comment:
If the string is constant, this would be better written with the substitution done, I'd think.  (The previous `mock` received two separate fields)

Reply:
The same comment applies to most of the other changes in this file; I'd also recommend using f-strings rather than `.format()` for these substitutions where there are variables; that's our standard approach in the codebase.

Reply:
It was told here(https://github.com/zulip/zulip/pull/15708#discussion_r461754030) in previous review to keep ```%s``` even if substitution is constant. Should I make it like this?
``` f"INFO:root:/complete/apple/: {"Authentication failed: Token validation failed"}"```